purchases/src/test/java/com/revenuecat/purchases/amazon/attribution/AmazonDeviceIdentifiersFetcherTests.kt

```diff
@@ -32,6 +34,12 @@ class AmazonDeviceIdentifiersFetcherTests {
         } returns 0
     }
 
+    @After
+    fun tearDown() {
+        unmockkStatic(Settings.Secure::class)
+        unmockkStatic(Log::class)
```

Comment:
Maybe we could just `clearAllMocks` which should also clear static mocks, but this is fine!